src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_params.py

```diff
@@ -126,48 +110,33 @@ def load_arguments(self, _):
         c.argument('policy', type=file_type, help='JSON encoded policy definition. Use the show command with JSON output to obtain a policy object. Modify the values using a file editor and pass the object.', completer=FilesCompleter())
 
     with self.argument_context('backup policy list') as c:
-        c.argument('vault_name', vault_name_type, id_part=None)
-
-    with self.argument_context('backup azure-wl policy') as c:
-        c.argument('vault_name', vault_name_type, id_part='name')
-
-    with self.argument_context('backup azure-wl policy list') as c:
-        c.argument('vault_name', vault_name_type, id_part=None)
         c.argument('workload_type', wl_type)
+        c.argument('container_type', backup_management_type)
 
-    with self.argument_context('backup azure-wl policy show') as c:
-        c.argument('name', policy_name_type, options_list=['--name', '-n'], help='Name of the backup policy. You can use the backup policy list command to get the name of a policy.')
+    with self.argument_context('backup policy show') as c:
+        c.argument('container_type', backup_management_type)
```

Comment:
Please use an appropriate value for the key.

Reply:
Need to discuss further to avoid clashes with ISVM implementation if any